classes/requests/post/class-dintero-checkout-create-session.php

```diff
@@ -75,6 +75,20 @@ public function get_body() {
 			'profile_id' => $this->settings['profile_id'],
 		);
 
+		// 'billing' => Default to customer billing address
+		// 'shipping' => Default to customer shipping address
+		// 'billing_only' => Force shipping to the customer billing address only.
+		$shipping_destination = get_option( 'woocommerce_ship_to_destination' );
+
+		$separate_shipping = wc_string_to_bool( $this->settings['express_allow_different_billing_shipping_address'] );
```

Comment:
This is a new setting right? So we should add a default fallback for the setting value to prevent issues with missing key

Reply:
Good catch! Updated.